network/network-tutorial-standalone.md

```diff
@@ -436,6 +436,10 @@ connected to both networks.
     address. This capability is called **automatic service discovery**. Let's
     connect to `alpine1` and test this out. `alpine1` should be able to resolve
     `alpine2` and `alpine4` (and `alpine1`, itself) to IP addresses.
+    
+    > **Note**
+    > 
+    > Only valid for custom container names, the default generated container names cannot be resolved
```

Comment:
@ChrisChinchilla , Thank you for asking me what I think.

It would be nice if the second half of the sentence emphasized that the name of the container is "automatically generated" or "generated by default". It would be more straightforward and would not require the reader to spend a few more seconds thinking "Generated? Does that mean automatically generated?"

That's what I thought.